libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/model/eh/EHorizonTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.mapbox.navigation.core.trip.model.eh
+
+import io.mockk.every
+import io.mockk.mockk
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+
+class EHorizonTest {
+
+    @Test
+    fun `current returns edge with the same edgeId`() {
+        val electronicHorizon = EHorizonTestUtil.loadLongBranch()
+        val eHorizon = EHorizonMapper.mapToEHorizon(electronicHorizon)
+
+        val edge = eHorizon.current(
+            mockk {
+                every { edgeId } returns 2452997120465L
+                every { percentAlong } returns 56.5
+            }
+        )
+
+        assertEquals(2452997120465L, edge.id)
+    }
+
+    @Test(expected = Exception::class)
+    fun `current throws when the edgeId does not exist`() {
```

Comment:
Only adding a couple tests to show that the `codecov` integration is working. Extensive testing on EHorizon is still coming